x/oracle/client/cli/queryOracle.go

```diff
@@ -0,0 +1,72 @@
+package cli
+
+import (
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/medibloc/panacea-core/v2/x/oracle/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdGetOracles() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "oracles",
+		Short: "Query a oracles info",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			pageReq, err := client.ReadPageRequest(cmd.Flags())
+			if err != nil {
+				return err
+			}
+
+			queryClient := types.NewQueryClient(clientCtx)
+
+			req := &types.QueryOraclesRequest{
+				Pagination: pageReq,
+			}
+			res, err := queryClient.Oracles(cmd.Context(), req)
+			if err != nil {
+				return err
+			}
+
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+
```

Comment:
```suggestion
	flags.AddQueryFlagsToCmd(cmd)
        flags.AddPaginationFlagsToCmd(cmd, "oracles")
```

The flags for pagination are missed

Reply:
Thank you!